aim/web/ui_v2/src/utils/toHash.ts

```diff
@@ -0,0 +1,25 @@
+import { encode, decode } from 'bs58check';
+
+export function bufferToString(buffer: Buffer): string {
+  return buffer.toString('utf-8');
+}
+
+export function contextToHash(context: { [key: string]: unknown }): string {
+  return encode(Buffer.from(JSON.stringify(context)));
+}
+
+export function traceToHash({
```

Comment:
Probably it is better to have one encoder which receives any object(s) and returns encoded string.

Reply:
yeah, definitely! @KaroMourad can you please add a separate utility function for encoding and use it in hashing functions?

Reply:
Fine , i will fix this